docs/commands/open.md

```diff
@@ -0,0 +1,61 @@
+# open
+
+Loads a file into a cell, convert it to table if possible.
```

Comment:
Looks good. You should also cover the `--raw` flag so people can learn about that, too.